index.html

```diff
@@ -0,0 +1,228 @@
+                        <li class="nav"><a href="#"><i class="bi bi-facebook"></i></a></li>
+                        <li class="nav"><a class="upper-nav-anchor" href="#"><span>My
+                                    Page</span></a></li>
+                        <li class="nav"><a href="#"><span>English</span></a></li>
+                        <li class="nav"><a href="#"><span>Login</span></a></li>
+                    </ul>
+                </nav>
+                <nav>
+                    <ul id="lower-nav" class="nav">
+                        <li><img src="images/logo.png" alt="Dev Courses Logo"
+                                width="70"></li>
+                        <li><a href="about.html">About</a></li>
+                        <li><a href="#">Older</a></li>
+                        <li><a href="#">Trending</a></li>
+                        <li><a href="#">Your Couses</a></li>
+                    </ul>
+                </nav>
+                <nav id="mobile-open-nav" class="display-none">
+                    <ul>
+                        <li id="close-button"><img
+                                src="images/Icon-close.png"
+                                alt="Close button"></li>
+                        <li><a href="#">Login</a></li>
+                        <li><a href="index.html">Home</a></li>
+                        <li><a href="#">My Page</a></li>
+                        <li><a href="#">Older</a></li>
+                        <li><a href="#">Trending</a></li>
+                        <li><a href="#">Your Couses</a></li>
+                        <li><a href="#"><i class="bi
+                                    bi-twitter"></i></a></li>
+                        <li><a href="#"><i class="bi bi-facebook"></i></a></li>
+                        <li><a href="/about.html">About</a></li>
+                        <li><a href="#">English</a></li>
+                    </ul>
+                </nav>
+                <section id="header-content">
+                    <h2 id="hello-header" class="orange-color">"Hello! Fellow
+                        Developers"</h2>
+                    <h1 class="orange-color">A LIST OF THE MOST USEFUL COURSES<br>
+                        FOR DEVELOPERS IN ALL
+                        FIELDS
+                        IN<br>
+                        2021</h1>
+                    <p>
+                        Browse through the website for the list of the greatest
+                        courses
+                        created in programming. Join our live webinars and start
+                        learning at
+                        your own pace with some of the highest quality courses
+                        ever
+                        created
+                        in programming.
+                    </p>
+                    <h1>2021.10.15(THU) ~ 16(FRI)</h1>
+                    <h3>Webinar on Web Development, Back End, Front End, and
```

Comment:
- [ ] The typography of the date and the location in the top section does not match the provided [template design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) Kindly improve upon this aspect by ensuring that the _font weight_, _font size_, _font color,_ and _line spacing_ replicates the provided design.

**Your top design (date and location):**
![image](https://user-images.githubusercontent.com/12745474/132544879-2605e853-923c-49bf-8b0b-b357df183529.png)

**The original top design (date and location):**
![image](https://user-images.githubusercontent.com/12745474/132544965-aa56ac66-2f08-4beb-9821-745a9b4f5fc5.png)